ext/recaptcha/CRM/Utils/ReCAPTCHA.php

```diff
@@ -62,31 +62,33 @@ public static function hasSettingsAvailable() {
    */
   public static function add(&$form) {
     $error = NULL;
+
+    // If statement has been altered so that we do not need to check !$this->get_template_vars("isCaptcha") in CRM/Contribute/Form/Contribution/Main.php
     if (!function_exists('recaptcha_get_html')) {
       require_once E::path('lib/recaptcha/recaptchalib.php');
-    }
```

Comment:
@kartik1000 Instead of moving everything into the if you could do this:

```
    if (function_exists('recaptcha_get_html')) {
      return;
    }

    require_once E::path('lib/recaptcha/recaptchalib.php');
    // ... rest of code
```